console.rb

```diff
@@ -3,15 +3,43 @@
 require_relative "models/tenant"
 
 # NOTE: Uncomment the lines below to verify that your seed script is working
-
+#
 # puts "There are #{Apartment.count} apartments"
 # puts "There are #{Tenant.count} tenants"
-#
 # puts "*" * 50
-#
 # puts "The first apartment is at #{Apartment.first.address}."
 # puts "It has  #{Apartment.first.tenants.count} tenants."
 
+value = 0
+while value < 5 do
+puts "Welcome, landlord! To see a list of all your apartments, press 1. \n
+To see a list of all your tenants, press 2.\n
+To see a list of apartments and their tenants, press 3"
+response = gets.chomp.to_i
+  if response == 1
+    puts Apartment.select('address, monthly_rent')
+  elsif
+    response == 2
+    puts Tenant.select('name, address')
+  elsif
+    response == 3
+    all_apartments = Apartment.all
+    all_apartments.each do
+      puts all_apartments.tenants
+    end
+    end
```

Comment:
There's an extra `end` keyword here that doesn't match with any block. This is creating a syntax error which is why running `ruby console.rb` was failing. One of these needs to be deleted.